_includes/master/reqs-sys.md

```diff
@@ -0,0 +1,31 @@
+## Node hosts
```

Comment:
I think "Node hosts" is a weird term. How about "Node requirements"

Reply:
FWIW I had that thought too, but wasn't sure what to suggest.